records_mover/records/sources/factory.py

```diff
@@ -52,25 +51,18 @@ def __init__(self,
     @contextmanager
     def dataframe(self,
                   df: 'DataFrame',
-                  schema_name: Optional[str]=None,
-                  table_name: Optional[str]=None,
-                  db_engine: Optional['Engine']=None,
                   processing_instructions: ProcessingInstructions=
                   ProcessingInstructions(),
                   records_schema: Optional[RecordsSchema]=None,
                   include_index: bool=False) -> Iterator['DataframesRecordsSource']:
         """
         :param df: Pandas dataframe to move data from.
-        :param schema_name: Obsolete - not used.
-        :param table_name: Obsolete - not used.
-        :param db_engine: Obsolete - not used.
         :param processing_instructions: Instructions used during creation of the schema SQL.
         :param records_schema: Description of the column names and types of the records.
         :param include_index: If true, the Pandas dataframe index column will be included in
         the move.
         """
 
-        warn_deprecated(schema_name=schema_name, table_name=table_name, db_engine=db_engine)
```

Comment:
I missed this in the big breaking release before!  This makes this a breaking PR.  The next release will also include the extras breaking change, so I figured we might as well.  I'll note this in the release notes and bump the middle release per semver, since we're still pre-1.0.  I'll also collect the internal uses in the corresponding internal ticket and we can figure out how to handle those.